.circleci/config.yml

```diff
@@ -409,9 +409,15 @@ jobs:
 
   deploy-dev:
     executor: docker-executor
+    parameters:
+      target-env:
+        type: string
+        default: 'raft'
```

Comment:
Should this default be left empty or does it fail if so? I noticed the default for target-env lower down is empty. Is there a use case where  the environment is not passed in and we need this pre-set?

Reply:
We could remove the default here, I had put this default in before updating the corresponding GitHub Action to pass in the target-env.